WELCOME.md

```diff
@@ -0,0 +1,47 @@
+👋 Hello @new_user_name and welcome to the For the Horde community! We're so happy you're here! The purpose of this community is to give you a space to connect with other Minnesota-based tech and tech-adjacent folks. Want to be successful here? Try to give more than you take.
+
+✅ First things first. This is a community and we want this to be a safe and inclusive space for everyone. We expect you to abide by the For the Horde Code of Conduct in this Slack team AND while participating in any in-person or online For the Horde events. Please head on over to https://git.io/v0d6b to check it out.
+
+Need to report Code of Conduct violation? The admins are listed at the bottom of the Code of Conduct. Snap a screenshot, report it.
+
+🚴🏿‍♀️🕺🏻👩‍💻😻⚾️✈️📺 Second, you've automagically been added to the #introductions channel. Please introduce yourself to the community there and share something fun about yourself. Where you're from, what you're working on, what your most peculiar hobby is, what your favorite color is, etc... We want to get to know you! Also, there's probably a channel for that peculiar hobby.
+
+📣 Speaking of channels. We have a lot. When thinking about where to post a message, go through the channel list and check there first. It’s very tempting with ~1200 eyeballs to post something to #general. We tend to treat #general like a watercooler, not a place to garner attention.
+
+_Popular channels:_
+
+- #frontend - talk about your javascript
+- #devs-n-ops - do not talk about your javascript
+- #cohordeworking - when, where, why, who
+- #business - general business topics
+- #mental-health - talk about it
+- #gigs - the ONLY place you can post/ask for jobs/gigs. They should be 100% available to Minnesota with no filter criteria, à la “might be interested in remote” Many people have notifications on for this channel, so 🧵any conversations beyond the initial post.
+
+_Fun, small channels:_
+
+- #art-hackers
+- #biking
+- #awww
+- #pets
+- #sports
+- #outdoors
```

Comment:
#gaming
#nerdparenting

Reply:
imo, these are by design a sampling and not exhaustive. The welcome message is already dangerously long and risks being ignored, so I would actually consider shortening these lists before adding to them...

Reply:
Not trying to be exhaustive, and I agree with you that this is already very long. But is two additional items in a bulleted list going to affect whether someone reads the whole thing or not? My guess is that they'll quickly scan the list regardless, and the presence of two additional topics not mentioned anywhere else in here _might_ encourage them to mention that they have kids or are gamers in their intro.

Reply:
Ok, I'm going to remove #outdoors and #pets and add #nerdparenting. #gaming, #pets, and #outdoors can be discoveries.